wiki/Skinning/fr.md

```diff
@@ -0,0 +1,32 @@
+# Skinning
+
+Le skinning est l'une des principales caractéristiques d'osu!. Cela permet aux joueurs de modifier les éléments de skinning originaux pour créer les leurs! Les skins peuvent varier, ils peuvent être amusants, thématisés ou alors être plus professionnels et sobres.
+
+## Skinning sets
+
+*Page principale: [Liste de skins](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+**Pour les beatmaps seulement,** si votre skin de beatmap contient un seul élément parmi les ensembles répertoriés dans les critères de classement, il doit contenir tous les autres éléments dudit ensemble. Cela ne s'applique qu'aux skins de beatmap.
+
+## Comment puis-je faire un skin
+
+*Page principale: [Tutoriel de skinning](/wiki/Skinning/Tutorial)*
+
+## Listes d'éléments de skins
+
+La liste des éléments de skinning est divisée en plusieurs catégories:
+
+- [Skinning de l'interface](/wiki/Skinning/Interface)
+- [Skinning osu!standard](/wiki/Skinning/osu!)
+- [Skinning osu!taiko](/wiki/Skinning/osu!taiko)
+- [Skinning osu!catch](/wiki/Skinning/osu!catch)
+- [Skinning osu!mania](/wiki/Skinning/osu!mania)
+- [Skinning sonore](/wiki/Skinning/Sounds)
+
+## Voir également
+
+- [skin.ini](/wiki/skin.ini)
+- [Histoire du skinning](/wiki/Skinning/History) - une liste d'anciens éléments de skinning
+- [Compendium](/community/forums/topics/686664) - une liste des skins terminées
```

Comment:
```suggestion
- [Compendium](https://osu.ppy.sh/community/forums/topics/686664) - une liste des skins terminées
```
only remove the hostname from `/wiki` links
could you change this in the other languages as well?